pytest_kubernetes/plugin.py

```diff
@@ -52,17 +54,21 @@ def remaining_clusters_teardown():
 
 
 def pytest_addoption(parser):
-    group = parser.getgroup("k8s")
-    group.addoption(
+    k8s_group = parser.getgroup("k8s")
+    k8s_group.addoption(
         "--k8s-cluster-name",
```

Comment:
Do we have a mutual exlusion of `--k8s-cluster-name`  and `--k8s-cluster-config` or precedence?